installers/flash-tarball/radiasoft-download.sh

```diff
@@ -14,36 +14,38 @@ flash_tarball_main() {
     local r=()
     install_url radiasoft/download installers
     install_script_eval rpm-code/codes.sh
-    codes_download rsflash
+    r="$(flash_untar $t)"
+    flash_patch "$r"
+    flash_add_examples "$r"
+    o=flash.tar.gz
+    tar czf "$o" "$r"
+    x=$d/flash-$(date -u +%Y%m%d.%H%M%S).tar.gz
+    rm -f "$x"
+    tar czf "$x" "$o"
+    ln -s --force "$(basename "$x")" $d/flash-dev.tar.gz
+}
+
+flash_add_examples() {
+    local flash_dir="$1"
+    codes_download flashcap
     git fetch --unshallow
     local x
     for x in \
-        'CapLaser3D 47a641aa467ff48c1337a69e6c3a6778e5b854ae' \
-        'CapLaserBELLA master' \
+        'CapLaser3D bella_3dSetup.par' \
+        'CapLaserBELLA bella.par' \
     ; do
         x=( $x )
-        git checkout --quiet "${x[1]}"
-        cd "config/${x[0]}"
-        # POSIT: Matches sirepo.sim_data._flash_problem_files_archive_basename
-        n=problemFiles-archive.${x[0]}.zip
-        zip --quiet "$p/$n" *F90 Config Makefile
-        r+=($n)
-        cd ../..
+        local i="config/${x[0]}"
+        # POSIT: sirepo.sim_data.flash.FLASH_PAR_FILE
+        cp "$i/${x[1]}" "$i/flash.par"
+        cp -R "$i"  "../$flash_dir/source/Simulation/SimulationMain/"
```

Comment:
What is the umask situation here? We probably don't want o+r because this is proprietary. Then again, it is inside the docker container for a specific user. In any event, it should be clear what we are doing and document it as such. This is why `install` is used, because it forces you to think about permissions.

Reply:
d146ff4